java-manta-client/pom.xml

```diff
@@ -170,6 +170,20 @@
             <version>${dependency.urlbuilder.version}</version>
         </dependency>
 
+        <!-- object cloning library for memoization of encryption state -->
```

Comment:
Do we want to shade and relocate these libraries or have the main library take it as a direct dependency?